README.md

```diff
@@ -3,7 +3,19 @@
 Discover current Supervisor Services offered to support modern applications through vSphere Services.  
 New service will be added overtime with the goal to continue to empower your DevOps communities.
 
-*Supervisor Services are only available with Supervisor Clusters enabled using NSX-T*
+Supervisor Services **are only** available with Supervisor Clusters enabled **using NSX-T**.
```

Comment:
Hmm that's not true though? PodVMs are supported on VDS (https://jira.eng.vmware.com/browse/VGL-1577) starting from 8.x (x=?) release, so we should beable to install services on VDS supervisors too.

Reply:
I haven't adjusted the original footnote since vSphere 8 Update 1 isn't GA'ed yet. Do you want the hint included already?
[Supervisor Services with vSphere Distributed Switch](https://core.vmware.com/resource/whats-new-vsphere-8-update-1#sec25426-sub4)

Reply:
@rguske Yes, since we announced it publicly already, I'd rather include it now?